backend_new/test/integration/ami-chart.e2e-spec.ts

```diff
@@ -0,0 +1,189 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from '../../src/app.module';
+import { PrismaService } from '../../src/services/prisma.service';
+import { jurisdictionFactory } from '../../prisma/seed-helpers/jurisdiction-factory';
+import { stringify } from 'qs';
+import { AmiChartQueryParams } from '../../src/dtos/ami-charts/ami-chart-query-params.dto';
+import { amiChartFactory } from '../../prisma/seed-helpers/ami-chart-factory';
+import { AmiChartCreate } from '../../src/dtos/ami-charts/ami-chart-create.dto';
+import { AmiChartUpdate } from '../../src/dtos/ami-charts/ami-chart-update.dto';
+import { IdDTO } from 'src/dtos/shared/id.dto';
+
+describe('AmiChart Controller Tests', () => {
+  let app: INestApplication;
+  let prisma: PrismaService;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    prisma = moduleFixture.get<PrismaService>(PrismaService);
+    await app.init();
+  });
+
+  const cleanUpDb = async (
+    amiChartIds: string[],
+    jurisdictionIds: string[],
+  ) => {
+    for (let i = 0; i < amiChartIds.length; i++) {
+      await prisma.amiChart.delete({
+        where: {
+          id: amiChartIds[i],
+        },
+      });
+    }
+    for (let i = 0; i < jurisdictionIds.length; i++) {
+      await prisma.jurisdictions.delete({
+        where: {
+          id: jurisdictionIds[i],
+        },
+      });
+    }
```

Comment:
Prisma has a way to delete many records: https://www.prisma.io/docs/concepts/components/prisma-client/crud#delete-multiple-records

Reply:
⚡

Reply:
@YazeedLoonat , does it make sense to update this logic to use `deleteMany`?